cip-0026.md

```diff
@@ -0,0 +1,55 @@
+    CIP: 26
+    Title: Extended Reed Solomon Address Format
+    Author: ohager, jjos
+    Comments-Summary: Adding Public Key in base36 to Reed-Solomon Address  
+    Status: Draft
+    Type: Enhancement
+    Created: 2021-03-26
+
+## Motivation
+
+From the very beginning, creating accounts in Burstcoin is not entirely intuitive, and the process suffers from the chicken-and-egg problem:
+For an account to be stored on the blockchain, there must be at least one transaction involving the named account.
+However, a problem arises from the theoretical possibility of a collision in Account ID or the Reed-Solomon representation, since the address space is smaller than that of the public key. Therefore, an account without a link to the public key is considered insecure vulgo inactive.
+For outgoing transactions, the public key is implicitly linked, but not necessarily for incoming transactions. But in order to execute an outgoing transaction, the owner needs some BURST to pay the transaction fees. To do this, the owner must somehow obtain BURST, e.g., ask for a small donation in the community, or use a working faucet service.
+Alternatively, an incoming transaction can include the public key along with the account id. In this case, the account will also be secured or activated.
+However, it is currently not user-friendly to get the public key. In this document a simplification of the account activation is described, which is based on an optional extension of the Reed-Solomon address.
+
+## Abstract
+
+The idea is to extend the Reed-Solomon address representation by the public key, such it can be implicitly linked with the public key.
+
+## Specification
+
+The account IDs can be identified by the Reed-Solomon encoding in the known form
+
+`BURST-XXXX-XXXX-XXXXX`
+
+In this proposal, it is suggested to simply append the public key to the Reed-Solomon address, resulting in the following format
+
+`BURST-XXXX-XXXX-XXXXX-YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY`
+
+Although technically readily defensible, there is an option to represent the public key, which is normally in hexadecimal form, in a more compact form.
+As Base64 encoding is case-sensitive, the Base36 encoding is used, resulting in a slightly more readable representation
+
+`BURST-XXXX-XXXX-XXXXX-ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ`
+
+
+_Real World Example_
+
+`BURST-K37B-9V85-FB95-793HN-2UCGWTUEEY66TN7RNC189PM19C4ATCEUGQV929IY1N24H0Y82Z`
+
```

Comment:
Maybe comment that the integrity of extended addresses can be easily checked, since with the extension you have the public key which should produce the same RS address prefix.